### DISCUSSION TITLE
SFLE real-time logging

### DISCUSSION THREAD

**Discussion by filipmelik** [wavelog/wavelog#819] [Ideas]
First, a little story :D I was activating FaunaFlora this weekend, using Wavelog's SFLE. It was the biggest pileup so far for me and I ended as 121 QSOs. So far so good, but I forgot to write time to QSOs (only the first QSO had time filled), so all my QSOs has the same time. lol!

Original SFLE software by DF3CB has an option to turn on "real time logging mode" which automatically adds current UTC time to each QSO automatically when you hit enter. This feature would help me a lot in this situation and it would certainly make using SFLE a little more easier.

This should be configurable in the SFLE options IMHO, you can either use it if you want or not.

WDYT? I think I could tackle this issue if there is an interest :)

**Comment by HB9HIL** [comment]
First a question: Why with SFLE and not with the normal QSO logging? 😄

**Reply by filipmelik** [reply]
At least three reasons at my mind now:

1) there was a shitty internet connection, and when on slow net, the Live QSO mode has some minor bugs like when it fires reques to qrz.com for station lookup, and the request comes with delay due to slow net, and in the meantime you rewite the call field, because you made a mistake, the fields get filled with the “old” data. This happened to me several times when logging on slow net.

2) when in the pileup, I find logging with SFLE much faster, because you do not have to use tab to switch between textfields. And also, when in pileup you usually do not really care about station lookup from qrz or hamqth. 

3) related to 2 above - if you would use Live QSO mode, you would exhaust your free qrz lookups (I am non subscriber) for nothing.

**Reply by HB9HIL** [reply]
You could use contest logging without exchange. 😄 

Point is that SFLE was never built for live QSO logging. Sure you can take a look at that. Be our guest

**Reply by HB9HIL** [reply]
PS: You are not the first one requesting this....

**Reply by filipmelik** [reply]
I just checked contest logging mode (never opened it before, because I have never been in contest) and on first sight it seems very overwhelming for my use case, which is outdoor portable activity: WWFF/POTA/GMA/SOTA...

I like the SFLE input for its input simplicity, even though it was never intended for this use case probably 😊 

> Sure you can take a look at that. Be our guest

I am asking first and discussing, since I do not want to waste time for something you might not want to merge for all the users (in case I am the only one who wants this, for example), but since you mentioned it was requested multiple times, I will try to tackle this 👍
